ui/src/app/pages/workspace/workspace-edit.tsx

```diff
@@ -1048,6 +1054,29 @@ export const WorkspaceEdit = fp.flow(
       this.saveWorkspace();
     }
 
+    private async apiCreateWorkspaceAsync() {
```

Comment:
this is the MVP UI, which the user should experience as having the same behavior as the synchronous version